packages/hello-gsm/src/pages/apply.tsx

```diff
@@ -4,16 +4,42 @@ import { GetApplicationType } from 'type/application';
 import application from 'Api/application';
 import { HeaderType } from 'type/header';
 import auth from 'Api/auth';
-import { ApplyPage } from 'PageContainer';
+import { ApplyPage, CalculatorPage } from 'PageContainer';
+import { useEffect, useState } from 'react';
 
 const Apply: NextPage<GetApplicationType> = ({ data }) => {
   const seoTitle = '입학 지원';
   const desc = '지원자의 인적사항을 기재합니다.';
 
+  const [step, setStep] = useState<'원서' | '성적'>('원서');
+
+  const preventClose = (e: BeforeUnloadEvent) => {
+    e.preventDefault();
+    e.returnValue = '';
+  };
+
+  useEffect(() => {
+    (() => {
+      window.addEventListener('beforeunload', preventClose);
+    })();
```

Comment:
```suggestion
  useEffect(() => {
      window.addEventListener('beforeunload', preventClose);
```

useEffect 안에 즉시실행함수 구문을 작성하지 않아도 되지 않을까요?

Reply:
지금 하고있는 작업이 있어 조금 후에 확인해보겠습니다! 리뷰 감사합니다.

Reply:
https://github.com/themoment-team/hello-gsm-front/pull/220/commits/0c36da1a567bd2064c71b81059e8239ac3a2ea0c

IIFE 문법을 제거하고, prevnetClose 함수를 useEffect 내부로 옮겼습니다! 좋은 지적 감사합니다.